src/compiler/commandLineParser.ts

```diff
@@ -120,6 +121,7 @@ namespace ts {
                     "es7": "lib.es2016.d.ts",
                     "es2016": "lib.es2016.d.ts",
                     "es2017": "lib.es2017.d.ts",
+                    "es2018": "lib.es2018.d.ts",

```

Comment:
you need to build the new library as well. this is defined in our Jake/gulp files (sorry we have two build systems for now untill we are done with the migration).
JakeFile: https://github.com/Microsoft/TypeScript/blob/8f1cdc9b0c649f1e4fe9d96718be93277baa2ecd/Jakefile.js#L229
GulpFle: https://github.com/Microsoft/TypeScript/blob/8f1cdc9b0c649f1e4fe9d96718be93277baa2ecd/Gulpfile.ts#L168
See https://github.com/Microsoft/TypeScript/commit/6a737c88f7734d7f6d70a85c11bf109bbe18436e that added esNext target.

Reply:
@mhegazy thank you for the review. I am updating my PR accordingly.

FYI, I see that jake LKG generates diagnosticMessages.generated.json file per locale. I think this is related to PR  #19308

Therefore, in my PR you will have 2 commits:
1. 'run jake LKG before es2018 addition' 
2. 'fix(es2018): add a target and an initial lib' (contains the es2018 lib + jake LKG run a second time)
Please let me know if I need to squash both commits